src/arbitrage.py

```diff
@@ -71,19 +74,73 @@ def get_underlying(self):
         return self.spot
 
     def get_option_prices(self):
-        """Returns a dict mapping expiry dates to a tuple (call, put)
-        prices for at-the-money options"""
+        """Returns a dict mapping expiry dates to
+        {"call": call_price, "put": put_price}"""
         return self.option_prices
 
 
 class API(object):
     def __init__(self, args=None):
         self.name = args
 
+    def get_stock_data(self, stock):
+        exps = self.get_expiries(stock)
+        spot = self.get_spot_price(stock)
+        premiums = {}
+        for exp in exps:
+            premiums[exp] = self.get_option_premiums(stock, exp, spot)
+        return StockData(stock, exps, spot, premiums)
+
+    def get_expiries(self, stock):
+        return
+
+    def get_spot_price(self, stock):
+        return
+
+    def get_option_premiums(self, stock, expiry, stock_price):
+        return
+
+
+class TradierAPI(API):
+    def __init__(self):
+        self.headers = {'Authorization': 'Bearer Bfo8MwBCA6lFOqWSdWIe1Ke7IigA', 'Accept': 'application/json'}
```

Comment:
put a TODO here to read the auth token from a config file

Reply:
#TODO(scott-xue): blah blah

Reply:
fixed